### PR TITLE
Shorter README.md for "post onboarding" branch.

### PR DESCRIPTION
**Pull request by john-sifive**
A briefer README file which 1) does NOT describe the steps in onboarding, and 2) describes the commands for testing, creating an FPGA bitstream and generating documents. It includes links to the "preonboarding" README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -619,63 +119,24 @@ bin, elf, and hex files of the test program can be found in
...
-If any sections are missing, they will be filled in with placeholder sections.
+The command creates two files, `pioDUT.html` and `pioDut.adoc`, both in the directory `build/api-generator-sifive/pioDUT/documentation`. 
+The .html file can be viewed directly in a web browser, and the .adoc file contains AsciiDoc which can be used for further processing,
+See the [AsciiDoctor PDF project](https://asciidoctor.org/docs/asciidoctor-pdf) for information on converting AsciiDoc to PDF.
 
 
 ## Wrap up
-Checkout tag `complete` if you would like to see how the repo should look
-like after completing the tutorial.
+Checkout tag [`preonboarded`](https://github.com/sifive/block-pio-sifive/blob/preonboarded/README.md)
```
Comment (albertchen-sifive) [inline README.md]: it's a branch not a tag

File: README.md [hunk 2]
```diff
@@ -40,6 +56,7 @@
     * See [verilator install instructions](https://www.veripool.org/projects/verilator/wiki/Installing)
     * Verilator additionally requires Perl v5.22.2
 * python3 and pip3
+* ruby
```
Comment (albertchen-sifive) [inline README.md]: what version of ruby? or does it not matter?

**Review by albertchen-sifive — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (john-sifive) [reply README.md]: Will do.

File: README.md (hunk 2 above)
Comment (john-sifive) [reply README.md]: I'll add "2.3.7 or later".  Thx.

File: README.md (hunk 1 above)
Comment (john-sifive) [reply README.md]: BTW, if "complete" is a tag, should "preonboarded" also be a tag?

Comment (albertchen-sifive) [reply README.md]: `complete` no longer exists in this repo. It was a tag on the old repo and we never updated the README when we copied it. Also I expect there will be changes to `preonboarded` in the future so I think it should be a branch.

**Review by richardxia — APPROVED** [review]
LGTM!

**Pull request merged**